.github/workflows/update_app_templates_hash.yml

```
@@ -0,0 +1,45 @@
+name: Update aws/aws-sam-cli with latest commit hash from aws/aws-sam-cli-app-templates
+
+on:
+  schedule:
+    - cron: "0 * * * *" # run at the top of every hour
+  workflow_dispatch: {}
+
+jobs:
+  updateCommitHash:
+    permissions:
+      pull-requests: write
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout App Templates
+        uses: actions/checkout@v3
+        with:
+          repository: aws/aws-sam-cli-app-templates
+          path: aws-sam-cli-app-templates
+
+      - name: Checkout SAM CLI
+        uses: actions/checkout@v3
+        with:
+          repository: aws/aws-sam-cli
+          path: aws-sam-cli
+
+      - name: Update hash & Raise PR for SAM CLI
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "GitHub Action"
+          cd aws-sam-cli-app-templates
+          APP_TEMPLATES_COMMIT_HASH=$(git rev-parse HEAD)
+          cd ../aws-sam-cli
+          git checkout -b update_app_templates_hash
+          git reset --hard develop
+          cat <<< $(jq --arg commit_hash "$APP_TEMPLATES_COMMIT_HASH" --indent 4 '.app_template_repo_commit =  $commit_hash' samcli/runtime_config.json) > samcli/runtime_config.json
```

Minor fix from https://github.com/koalaman/shellcheck/wiki/SC2046

```suggestion
          cat <<< "$(jq --arg commit_hash "$APP_TEMPLATES_COMMIT_HASH" --indent 4 '.app_template_repo_commit =  $commit_hash' samcli/runtime_config.json)" > samcli/runtime_config.json
```